graph/api/src/main/scala/org/apache/spark/graph/api/PropertyGraph.scala

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.graph.api
+
+import org.apache.spark.sql.{Dataset, Row}
+
+/**
+ * A Property Graph as defined by the openCypher Property Graph Data Model.
+ *
+ * A graph is always tied to and managed by a [[CypherSession]].
+ * The lifetime of a graph is bound by the session lifetime.
+ *
+ * @see <a href="http://www.opencypher.org/">openCypher project</a>
```

Comment:
Hmm. I understand the history, but the site itself announces a false news.
According to the our dev mailing list, there is no evidence for the followings.
> Databricks and Neo4j contributors have proposed to integrate the openCypher language into Apache Spark 3.0 to become a central piece of a new graph module in Apache Spark 3.0, driven through the openCypher project Cypher for Apache Spark

@mengxr already told us that he is an individual to work on this. So, if you want to keep this link, you must remove the announce on the site. In general, I know that it's beyond of this PR scope, so, you can remove this link from the PR for now.

Reply:
I will also make a note of changing the message on that webpage.

Reply:
Thanks. Than, it's okay! We can keep it.